Plutarch/DataRepr/Internal.hs

```diff
@@ -178,6 +214,16 @@ instance PIsData (PDataSum defs) where
   pdata = punsafeCoerce
   pfromData = punsafeCoerce
 
+instance PEq (PDataSum defs) where
+  x #== y = pdata x #== pdata y
+
+instance MkLtReprHandler defs => POrd (PDataSum defs) where
```

Comment:
You can make a more efficient instance for sums with only one constructor.

Reply:
I think that case should be handled by [`pmatchLT`](https://github.com/Plutonomicon/plutarch/pull/371/files#diff-850ef084fc92b4778569ed7b35931834f5de4efd686a7d6fd242ec21d19c503fR397) regardless